tests/unit/amo/components/TestAddonsByAuthorsCard.js

```diff
@@ -347,6 +350,26 @@ describe(__filename, () => {
     sinon.assert.notCalled(dispatchSpy);
   });
 
+  it('should have extensions length equal to numberOfAddons props', () => {
+    const root = renderAddonsWithType({
+      addonType: ADDON_TYPE_EXTENSION,
+      multipleAuthors: false,
+      numberOfAddons: 4,
+    });
+
+    expect(root.find(AddonsCard).props().addons).toHaveLength(4);
+  });
+
+  it('should have themes length equal to numberOfAddons props', () => {
+    const root = renderAddonsWithType({
+      addonType: ADDON_TYPE_THEME,
+      multipleAuthors: false,
+      numberOfAddons: 6,
```

Comment:
This is the default no?

Reply:
ok then could you please use something different than the default value?

Reply:
done